### ISSUE TITLE
Error in eco chapter

### ISSUE THREAD

**Issue by Robinlovelace** [geocompx/geocompr#1118]
```
62/70 [15-eco-27]        
Error in `.__AutoTuner__validate()`:
! could not find function ".__AutoTuner__validate"
Backtrace:
  1. base::eval(...)
  2. base::eval(...)
  4. rmarkdown (local) `<fn>`(...)
  5. knitr::knit(knit_input, knit_output, envir = envir, quiet = quiet)
  6. knitr:::process_file(text, output)
     ...
 17. base::withRestarts(...)
 18. base (local) withRestartList(expr, restarts)
 19. base (local) withOneRestart(withRestartList(expr, restarts[-nr]), restarts[[nr]])
 20. base (local) docall(restart$handler, restartArgs)
 22. evaluate (local) fun(base::quote(`<smplErrr>`))
Warning messages:
1: In warn_deprecated("Task$data_formats") :
  Task$data_formats is deprecated and will be removed in the future.
2: In warn_deprecated("DataBackend$data_formats") :
  DataBackend$data_formats is deprecated and will be removed in the future.
3: In warn_deprecated("Learner$data_formats") :
  Learner$data_formats is deprecated and will be removed in the future.
```

Source: https://github.com/geocompx/geocompr/actions/runs/11008562689/job/30566434904#step:4:2823

Any ideas @jannes-m or @Nowosad ?

**Comment by Robinlovelace** [comment]
All these lines seem to be failing, I'm trying to re-run them:

https://github.com/geocompx/geocompr/blob/5855dea6b42320c5531266feed8c155463b819ac/15-eco.Rmd#L508-L534

**Comment by Robinlovelace** [comment]
Attempted fix: https://github.com/geocompx/geocompr/pull/1117/commits/737cbfcd0be08fbe2bfb0161dc9b08ce2fa00ad4

**Comment by jannes-m** [comment]
closed via #1124
